client/my-sites/sidebar/sidebar.jsx

```diff
@@ -507,15 +507,9 @@ export class MySitesSidebar extends Component {
 		/* eslint-disable wpcalypso/jsx-classname-namespace*/
 		return (
 			<li className="wp-admin">
-				<a
-					onClick={ this.trackWpadminClick }
-					href={ adminUrl }
-					target="_blank"
-					rel="noopener noreferrer"
-				>
+				<a onClick={ this.trackWpadminClick } href={ adminUrl }>
```

Comment:
We might use `ExternalLink` here, we don't need to have it open in another window:

https://github.com/Automattic/wp-calypso/blob/master/client/components/external-link/index.jsx